### DISCUSSION TITLE
React navigation Modal with ScrollView inside

### DISCUSSION THREAD

**Discussion by Hantex9** [react-navigation/react-navigation#10668] [Q&A]
Hello everyone
I have a problem with the modal-presentation of react-navigation v6 to present a modal with ScrollView inside of it.
Actually if I enable gesture to swipe down the modal, it doesn't work if there is a ScrollView inside and I can swipe down only on the header.

I need that when reaching the ScrollView's top, must start dismiss animation gesture, so I need to reach the attached behavior;
This is the expected behavior:

![preview](https://user-images.githubusercontent.com/34343582/176510808-768f6227-7390-4444-a617-c3d0670117ee.gif)

Do you know how can i reproduce this without any external library but only using react-navigation v6?

**Comment by ShchipskiiYura** [comment]
I have same problem using react-navigation v6
